Gems/OpenXRVk/Code/Source/OpenXRVkDevice.cpp

```diff
@@ -91,12 +91,17 @@ namespace OpenXRVk
         }
 
         // Now that we have created the device, load the function pointers for it.
+#if AZ_TRAIT_OS_IS_HOST_OS_PLATFORM
+        bool functionsLoaded = xrVkInstance->GetFunctionLoader().LoadProcAddresses(
+            &xrVkInstance->GetContext(), xrVkInstance->GetNativeInstance(), xrVkInstance->GetActivePhysicalDevice(), m_xrVkDevice);
+#else
```

Comment:
Instead of repeating the whole line, you could extract only the part that's different and wrap it under the OS trait.

```cpp
VkPhysicalDevice vkPhysicalDevice = VK_NULL_HANDLE;

#if AZ_TRAIT_OS_IS_HOST_OS_PLATFORM
    // ...
    vkPhysicalDevice = xrVkInstance->GetActivePhysicalDevice();
#endif

// ...
bool functionsLoaded = xrVkInstance->GetFunctionLoader().LoadProcAddresses(
    &xrVkInstance->GetContext(), xrVkInstance->GetNativeInstance(), vkPhysicalDevice, m_xrVkDevice);
```